### ISSUE TITLE
Insufficient colour contrast of first item in dropdown menus

### ISSUE THREAD

**Issue by michael-milette** [michael-milette/moodle-theme_gcweb#9]
Here are a couple of examples from the Dashboard:
![image](https://user-images.githubusercontent.com/3808579/122858200-e2d7bd80-d2e7-11eb-845d-c5bbcdbfb049.png)
![image](https://user-images.githubusercontent.com/3808579/122858253-fd119b80-d2e7-11eb-9aec-1b9ff20133c0.png)

**Comment by michael-milette** [comment]
Fixed.
